kerastuner/engine/base_tuner.py

```diff
@@ -287,8 +287,6 @@ def results_summary(self, num_trials=10):
         Args:
             num_trials (int, optional): Number of trials to display.
                 Defaults to 10.
-            sort_metric (str, optional): Sorting metric, when not specified
-                sort models by objective value. Defaults to None.
         """
         display.section('Results summary')
```

Comment:
Does this still work?

Reply:
Deleted the docstring because it looks like `sort_metric` isn't used anymore.